site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -620,7 +646,82 @@ components:
          description: Allows arbitrary properties as extension to the ones
                       specified by the Hono API.
 
-# Tenant schema
+
+      PagingSortingFilteringOptions:
+         type: object
+         additionalProperties: false
+         properties:
+            "page":
+               $ref: '#/components/schemas/PagingOptions'
+            "filters":
+               $ref: '#/components/schemas/FilterOptionsList'
+            "sort":
+               $ref: '#/components/schemas/SortingOptions'
```

Comment:
this refers to a non-existing type. Should probably also be
```yaml
"sort":
  type: array
  items:
    $ref: '#/components/schemas/SortingOption'
```
and then we can remove `SortingOptionsList`